std_msgs/Header header
bool light_on
# relative luminance (Digital ITU BT.601)
# relative luminance is not illuminance and not in Lux unit
float32 relative_luminance
